docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/listing/list_item_macros.twig

```diff
@@ -225,15 +225,14 @@
     <span class="media-type">({{ media_link_type|trans }})</span>
   {% endif %}
   {% if media_link_type == "File" %}
-    <span class="filetype {{file_type}}">
+    <span class="filetype">
       {% if file_type != "unknown" %}
-        {# Note, the spanish should have the word order reversed... #}
-        <span class="show-for-sr">{{ file_type }} {% trans %}file{% endtrans %}</span>
+        [{{file_type}}
       {% else %}
-        {{ file_ext }}<span class="show-for-sr"> {% trans %}file{% endtrans %}</span>
+        [{{ file_ext }}
       {% endif %}
     </span>
-    <span class="filesize">({{ file_size }})</span>
+    <span class="filesize">, {{ file_size }}]</span>
```

Comment:
Can you rework this a bit for readability? e.g.
```
{% set filetype_text = (file_type != "unknown") ? file_type : file_ext %}
[<span class="filetype">{{filetype_text}}</span>, <span class="filesize">, {{ file_size }}</span>]
```

Also, while I am reading this, the classnames should all follow BEM naming too. Taking this thinking one step further, does there need to be a difference between the file type and file size? Previously yes because of background image css, now, I don't think so.